File: Python/no. checking/factor.py
#Factor of a no.
n=int(input("Enter the no. = "))
i=1
print("Factor of ",n,"is ",end=" = ")
while(n!=1):
    if n%i==0:
        print(i,end=" ")
        n//=i
        if i==1:
            i+=1
    else:
        i+=1

File: C Programms/for loop/factor.c
//WAP to check the factor of a no.
#include<stdio.h>
void main()
{
    int n,i;
    printf("Enter the no. = ");
    scanf("%d",&n);
    if (n>0)
    {
        printf("Factor of %d = ",n);
        for (i=1;i<=n;i++)
        {
            if (n%i==0)
                printf("%d ",i);
        }
    }
    else
        printf("Invalid Input!!");
}